src/storage/storage.ts

```diff
@@ -326,24 +333,52 @@ export class Storage {
 
     if (repo && !repo.eventsAttached) {
       log('Subscribing to channel ', channelAddress)
-      db.events.on('write', (_address, entry) => {
-        log('Writing to messages db')
-        log(entry.payload.value)
-        socketMessage(this.io, { message: entry.payload.value, channelAddress })
-      })
-      db.events.on('replicated', () => {
-        log('Message replicated')
-        loadAllMessages(this.io, this.getAllEventLogEntries(db), channelAddress)
-      })
-      db.events.on('ready', () => {
+      if (!this.isWaggleMobileMode) {
```

Comment:
This is not pretty but I am assuming that it's temporary. Could you add a note to README what isWaggleMobileMode is for and that this will be removed?